app/views/recipe_ingredients/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class='container wrapper'>
+  <div class='row'>
+    <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
+      <table class="table table-sm">
+        <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Amount</th>
+            <th scope="col"></th>
+          </tr>
+        </thead>
+        <%= form_for :recipe_ingredients, url: {action: 'create'} do |rec_ing| %>
```

Comment:
I think that we should create our form with `@recipe_ingredient` which we define in controller.